packages/liferay-npm-scripts/src/scripts/build.js

```diff
@@ -100,6 +100,8 @@ module.exports = function() {
 
 	runBundler();
 
+	translateSoy(BUILD_CONFIG.output);
```

Comment:
I know that utils/soy.js already has access to the `BUILD_CONFIG` (for example, `buildSoy()` uses it), but I prefer to pass this explicitly so that the function can be tested.

Haven't actually written tests though because this is hopefully going to be a short lived code path. Maybe the guilt will overcome me and I'll write some tests later.